td.vue/src/i18n/index.js

```diff
@@ -10,6 +10,7 @@ import deu from './de.js';
 import ell from './el.js';
 import eng from './en.js';
 import id from './id.js';
+import ms from './ms.js';
```

Comment:
ideally both id.js and ms.js would be imported in alphabetical order, so could these both be moved to after hi.js